modules/pytket-cirq/pytket/extensions/cirq/cirq_convert.py

```diff
@@ -22,7 +22,7 @@
 from logging import warning
 from cirq.devices import LineQubit, GridQubit
 import cirq.ops
-import cirq_google
+import cirq_google  # type: ignore
```

Comment:
Here as well, I'm just confused why the type checks had to be changed. Is it that Cirq has newly introduced type annotations?